astropy/convolution/tests/test_discretize.py

```diff
@@ -2,15 +2,15 @@
 
 import itertools
 
-import pytest
 import numpy as np
+import pytest
 from numpy.testing import assert_allclose
```

Comment:
Just to be difficult: I'm not a fan of this sorting, as I like to keep the Numpy imports together :)

Reply:
I can add numpy as it's own section if you want?